inc/PassHash.class.php

```diff
@@ -72,7 +91,7 @@ function verify_hash($clear, $hash) {
         } elseif(preg_match('/^\$6\$(rounds=\d+)?\$?(.+?)\$/', $hash, $m)) {
             $method = 'sha512';
             $salt   = $m[2];
-			$magic  = $m[1];
+            $magic  = $m[1];
```

Comment:
Is there a tab in here? Or does it fix a tab? I cannot tell from the diff.

Reply:
There was some tabs in the file. They've all been converted to space since the file mostly uses space. Here, there was a tab.